pkg/test/framework/features/whitelist.txt

```diff
@@ -0,0 +1,25 @@
+tests_setup_profile_default, TestDocs/ops/configuration/mesh/app-health-check/test.sh
+tests_setup_profile_default, TestDocs/tasks/security/authentication/authn-policy/test.sh
+tests_setup_profile_default, TestDocs/tasks/security/authorization/authz-deny/test.sh
+tests_setup_profile_default, TestDocs/tasks/security/authorization/authz-http/test.sh
+tests_setup_profile_default, TestDocs/tasks/security/authorization/authz-ingress/test.sh
+tests_setup_profile_default, TestDocs/tasks/security/authorization/authz-jwt/test.sh
+tests_setup_profile_default, TestDocs/tasks/security/authorization/authz-tcp/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/circuit-breaking/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/egress/egress-tls-origination/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/fault-injection/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/ingress/ingress-control/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/ingress/ingress-sni-passthrough/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/ingress/secure-ingress/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/mirroring/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/request-routing/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/request-timeouts/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/tcp-traffic-shifting/test.sh
+tests_setup_profile_default, TestDocs/tasks/traffic-management/traffic-shifting/test.sh
+tests_setup_profile_demo, TestDocs/examples/bookinfo/test.sh
+tests_setup_profile_demo, TestDocs/tasks/observability/logs/access-log/test.sh
+tests_setup_profile_demo, TestDocs/tasks/security/authentication/mtls-migration/test.sh
+tests_setup_profile_demo, TestDocs/tasks/traffic-management/egress/egress-gateway-tls-origination/mtls_test.sh
+tests_setup_profile_demo, TestDocs/tasks/traffic-management/egress/egress-gateway-tls-origination/tls_test.sh
+tests_setup_profile_none, TestDocs/tasks/security/cert-management/dns-cert/test.sh
+tests_setup_profile_none, TestDocs/tasks/security/cert-management/plugin-ca-cert/test.sh
```

Comment:
We need to turn this off for all, not list the tests one by one.

Reply:
Unfortunately, there isn't an option right now. Someone like @therealmitchconnors would have to add it to the framework.

Reply:
Some option we can add when running to not output the message and the fatalf.

Reply:
Another weird item I found was I had to have a slightly different suite name (adding a _work) when trying to run locally.

Reply:
How about creating a new feature, `documentation`, and add a call to `Features` [after this line](https://github.com/istio/istio.io/blob/a7382bef813024cb3f9d1c95b898987578cf761a/pkg/test/istioio/framework.go#L193) to make them all implement that feature.

Reply:
I think that works as well.